FCO.tex

```diff
@@ -105,60 +105,59 @@ \section{Fix mixing ratio vs Pu-equivalent theory}
   \caption{Evolution of the plutonium content in the J1'' and  E3'' stream\label{fig:MW_flow} }
 \end{figure}
 
-The difference on J1'' and E3'' between all 6 calculations can be observed on
-Figure \ref{fig:MW_flow}. First one should only consider in this study the time
-after 15 and 75y, as the calculation need almost 12y to rich an equilibrium. 
+The difference between J1'' and E3'' for all 6 calculations can be observed in
+Figure \ref{fig:MW_flow}. Note that for this study one should only consider the time
+after 15 and 75y, as the calculation needs almost 12y to reach an equilibrium. 
```

Comment:
I have should removed the 75yr mention (it is an old one... I believed I did... sorry)